data/blog/vitor-pamplona-receives-lts-grant.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Long-Term Support For Vitor Pamplona"
+date: '2024-08-14'
+tags: ['OpenSats', 'grants', 'nostr', 'LTS']
+draft: false
+authors: ['arvin', 'dergigi']
+images: ['/static/images/blog/47-vitor-pamplona.jpg']
+summary: "OpenSats Welcomes Vitor Pamplona as an LTS Grantee."
+---
+
+OpenSats is proud to announce that Vitor Pamplona has been granted long-term
+support for his exceptional contributions to the nostr ecosystem.
+
+[Vitor](https://github.com/vitorpamplona) is the creator of Amethyst, a
+widely-used and configurable nostr client for Android users, and one of the
+architects behind the [new hyper-private DM
+protocol](https://github.com/nostr-protocol/nips/blob/master/17.md) for nostr.
+Among its [extensive features](https://github.com/vitorpamplona/amethyst#),
+Amethyst supports multiple user accounts, live streams, marketplaces, group
+chats, custom feeds via data vending machines, and integrates bitcoin lightning
+"zaps" to enable users to exchange value directly within the app while
```

Comment:
NIT: I'd end the sentence after "withing the app." -- and mention Tor support in a new sentence. (It's a very long sentence.)

Reply:
I second this. It's a tad long. :)

Reply:
Thank you guys 🙏 

Edited to the following:

> Privacy and security are further enhanced by allowing traffic routing through the Tor network via [Orbot](https://orbot.app/en/).